lib/hal/video.c

```diff
@@ -198,7 +198,13 @@ BOOLEAN XVideoListModes(VIDEO_MODE *vm, int bpp, int refresh, void **p)
 	
 	DWORD dwAdapter = dwEnc & 0x000000FF;
 	DWORD dwStandard = dwEnc & 0x0000FF00;
-	
+
+        bool is_480p;
+        bool allow_480p = dwEnc & VIDEO_MODE_480P;
+        bool allow_480i = !allow_480p;
+        bool allow_720p = dwEnc & VIDEO_MODE_720P;
+        bool allow_1080i = dwEnc & VIDEO_MODE_1080I;
+
```

Comment:
Indentation is different from surrounding code

Reply:
Fixed